tests/foreman/ui/test_myaccount.py

```diff
@@ -15,24 +15,90 @@
 
 @Upstream: No
 """
+import random
 
+from fauxfactory import gen_string
+from nailgun.entities import User
+from robottelo.constants import LANG_TO_LOCALE_DCT
 from robottelo.decorators import stubbed, tier1
 from robottelo.test import UITestCase
+from robottelo.ui.locators import common_locators
+from robottelo.ui.locators import locators
+from robottelo.ui.my_account import MyAccount
+from robottelo.ui.session import Session
+
+
+def _valid_string_data(min_len=1, max_len=50):
+    """Generates valid string data for ui tests
```

Comment:
Can't we use valid_data_list from https://github.com/SatelliteQE/robottelo/blob/master/robottelo/datafactory.py#L138